driver-core/src/main/java/com/datastax/driver/core/querybuilder/QueryBuilder.java

```diff
@@ -277,6 +277,26 @@ public static Clause eq(String name, Object value) {
     }
 
     /**
+     * Creates an "equal" where clause for a group of clustering columns.
+     * <p/>
+     * For instance, {@code eq(Arrays.asList("a", "b"), Arrays.asList(2, "test"))}
+     * will generate the CQL WHERE clause {@code (a, b) = (2, 'test') }.
+     * <p/>
+     * Please note that this variant is only supported starting with Cassandra 2.0.6.
+     *
+     * @param names  the column names
+     * @param values the values
+     * @return the corresponding where clause.
+     * @throws IllegalArgumentException if {@code names.size() != values.size()}.
+     */
+    public static Clause eq(List<String> names, List<?> values) {
```

Comment:
The existing methods (`lt`, `gt`, etc.) didn't, I'd rather avoid overloading the API.